docs/mobile.md

```diff
@@ -0,0 +1,80 @@
+# Mobile local development
+
+Instructions to use an iPhone or Android mobile device for local app development, in two sections:
+* [§ Use the app from a mobile device](#use-the-app-from-a-mobile-device) by running it over your home or office WiFi
+* [§ Debugging with the desktop browser](#debugging-with-the-desktop-browser) by plugging your phone into your computer with a USB cable
+
+## Use the app from a mobile device
+
+These instructions will configure your local copy of the identity-idp app to serve web pages over your local computer network &mdash; the wifi in your home or office. You can broadcast the app to a mobile phone or tablet. Both your mobile device and your development computer (your laptop) must be connected to the same wifi network.
+
+By default, the application binds to `localhost`. These instructions bind to `0.0.0.0` instead. Some Android users report they can access `localhost:3000` directly on their phone, however.
+
+1. Find your Local Area Network IP address. On a MacBook, this is available at **System Preferences → Network**. The address may start with `192.168`.
+
+2. To your app's `application.yml` file, add the below. Fill in your actual LAN IP address. The final line creates a **confirm now** link in place of email confirmation.
+```yaml
+domain_name: 192.168.x.x:3000
+mailer_domain_name: 192.168.x.x:3000
+enable_load_testing_mode: true
+```
+
+3. In the Chrome web browser of your development computer, visit `chrome://inspect`
```

Comment:
I wonder if it's worth breaking these instructions down based on iOS vs. Android, since...

1. On iOS, it's not necessary to configure port forwarding for Chrome
2. On Android, once you set up port forwarding, you can navigate to localhost on your phone and you don't need to change the `application.yml` `domain_name`, which is much more convenient

Reply:
Yeah, it's probably worth it.

Reply:
This isn't matching my experience; I'm not able to get to `localhost` on my Android device. I'm going to merge this for now; it can be revisited later.